sdk/documentintelligence/ai-document-intelligence-rest/CHANGELOG.md

```diff
@@ -21,6 +21,78 @@
         throw new Error("The response was unexpected.");
       }
   ```
+- New `parseOperationIdFromResponse` method has been added to parse `operationId` from the initial response of batch analysis requests which can later be used to get the analysis results.
+  ```js
+      // Example
+      const initialResponse = await client
+        .path("/documentModels/{modelId}:analyzeBatch", "prebuilt-layout")
+        .post({
+          contentType: "application/json",
+          body: {
+            azureBlobSource: {
+              containerUrl: batchTrainingFilesContainerUrl(),
+            },
+            resultContainerUrl: batchTrainingFilesResultContainerUrl(),
+            resultPrefix: "result",
+          },
+        });
+
+      if (isUnexpected(initialResponse)) {
+        throw initialResponse.body.error;
+      }
+      const batchOperationId = parseOperationIdFromResponse(initialResponse);
```

Comment:
Dont know if it impacts JS, but we are exposing the operationId as resultId in Java.

Reply:
is that a rename?

Reply:
Here's what Paul said during archboard:
> OperationDetails.operationId is for :build, :compose, and :copyTo operations.  There, operationId is more appropriate than resultId.  AnalyzeBatchResultOperation and AnalyzeResultOperation are LRO :analyze.  Here, resultId is more semantically accurate.

I'm not sure if it was named differently before, but the property is defined as `resultId` in the spec today.

Reply:
Updated the global method to `parseResultIdFromResponse`